MIDAS/src/hardware/GPSSensor.cpp

```diff
@@ -0,0 +1,27 @@
+#include "sensors.h"
+#include "pins.h"
+#include "MicroNMEA.h"
+#include "teseo_liv3f_class.h"
+#include "sensor_data.h"
+#include <Wire.h>
+
+// Replace "Wire" with a reference to the I2C object to be used by the sensor
+TeseoLIV3F teseo(&Wire, GPS_RESET, GPS_ENABLE);
+
+ErrorCode GPSSensor::init() {
+    teseo.init();
```

Comment:
should we be capturing the error here and doing something in the event that it isn't successful

Reply:
Good idea